tests/testsuite/build.rs

```diff
@@ -1024,12 +1024,12 @@ fn cargo_compile_with_downloaded_dependency_with_offline() {
         p2.cargo("build")
             .masquerade_as_nightly_cargo()
             .arg("-Zoffline"),
-        execs().with_stderr(format!(
+        execs().with_stderr(
             "\
 [COMPILING] present_dep v1.2.3
 [COMPILING] bar v0.1.0 ([..])
-[FINISHED] dev [unoptimized + debuginfo] target(s) in [..]"
-        )),
+[FINISHED] dev [unoptimized + debuginfo] target(s) in [..]".to_string()
```

Comment:
these `to_string()`'s aren't necessary as `with_stderr` takes `S: ToString`, and thus can take a string slice.

Reply:
Thanks, updated.